animations/crowdsource/main.py

```diff
@@ -0,0 +1,139 @@
+import sys
+from oscapi import ColorsOut
+from animations import FadeAnimation 
+import imaplib
+import getpass
+import email
+import email.header
+import datetime
+import random
+from random import randint
+import time
+
+EMAIL_ACCOUNT = "lightschroma@gmail.com"
+EMAIL_FOLDER = "INBOX"
+
+black = (0.0,0.0,0.0)
+green = (0.0, 1023.0, 0.0)
+red = (1023.0, 0.0, 0.0)
+blue = (0.0, 0.0, 1023.0)
+orange = (1023.0, 511.5,0.0)
+yellow = (1023.0, 1023.0, 0.0)
+purple = (1023.0, 0.0, 1023.0)
+teal = (511.5, 1023.0, 1023.0)
+
+pixels = [black,black,black,black,black,black,
+black,black,black,black,black,black,
+black,black,black,black,black,black,
+black,black,black,black,black,black,
+black,black,black,black,black,black,
+black,black,black,black,black,black,
+black,black,black,black,black,black,
+black,black,black,black,black,black]
+
+if __name__ == "__main__":
+    password = "PASSWORD GOES HERE"
```

Comment:
you can put the password in a file, identified by an environment variable. this way, you're not modifying scripts on a production machine.